android/build.gradle

```diff
@@ -59,14 +59,15 @@ dependencies {
   implementation "com.android.installreferrer:installreferrer:${safeExtGet('installReferrerVersion', '1.1.2')}"
   def firebaseBomVersion = safeExtGet("firebaseBomVersion", null)
   def firebaseIidVersion = safeExtGet('firebaseIidVersion', null)
+  def googlePlayServicesIidVersion = safeExtGet('googlePlayServicesIidVersion', null)
+
   if (firebaseBomVersion) {
       implementation platform("com.google.firebase:firebase-bom:${firebaseBomVersion}")
       implementation "com.google.firebase:firebase-iid"
```

Comment:
@mikehardy hi, after setting `firebaseBomVersion` on build.gradle I get:

```
Could not determine the dependencies of task ':react-native-device-info:compileDebugJavaWithJavac'.
> Could not resolve all dependencies for configuration ':react-native-device-info:debugCompileClasspath'.
   > Could not find com.google.firebase:firebase-iid:.
     Required by:
         project :react-native-device-info
```

android/build.gradle:

```
buildscript {
    ext {
        googlePlayServicesIidVersion = "17.0.0"
        firebaseBomVersion = "33.14.0"
    }
```

i believe that if we set `firebaseBomVersion` we should also set the `FirebaseIidVersion` too, right?